### PR TITLE
Add a guide on using wolfictl to create and update advisories

### PR DESCRIPTION
**Pull request by mcaveety**
@luhring sorry to ping you a million times on this effort! I pulled this article off the other PR on the Academy so it could go here instead where it might be more useful. I added it to the docs folder since that seemed most appropriate. I do imagine it could use some additional edits and changes to suit this location better but I figure I'll hand it off?

resolves #https://github.com/wolfi-dev/advisories/issues/91

If there is anything you'd like me to change i'd be happy to!

### CODE REVIEW

File: docs/managing-advisories.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+First, we will be asked to enter the package in which the CVE was found. For our demonstration we will use the `glibc` package. Then, we will be asked for the vulnerability ID that we wish to make an advisory for. This ID must be a CVE, CGA, GHSA, or Go vulnerability ID. Finally, we will be asked what the status for the advisory should be, from one of the following options:
+- detection (Under investigation)
+- true-positive-determination (Affected)
+- fixed (Fixed)
+- false-positive-determination (Not affected)
+- fix-not-planned (Fix not planned)
+- pending-upstream-fix (Pending upstream fix)
+
+To learn more about the meanings of each of these identifiers, please refer to our [documentation on event types](https://github.com/wolfi-dev/advisories/blob/main/docs/event_types.md). The following shows an example of the `wolfictl advisory create` command in action. **Please note that the vulnerability exemplified here is an arbitary CVE ID for demonstration purposes and does not reflect an actual vulnerability found in the `glibc` package.**
```
Comment (luhring) [inline docs/managing-advisories.md:115]:
```suggestion
To learn more about the meanings of each of these identifiers, please refer to our [documentation on event types](https://github.com/wolfi-dev/advisories/blob/main/docs/event_types.md). The following shows an example of the `wolfictl advisory create` command in action. **Please note that the vulnerability exemplified here is an arbitrary CVE ID for demonstration purposes and does not reflect an actual vulnerability found in the `glibc` package.**
```

File: docs/managing-advisories.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  fixed
+Fixed Version: 2.39-r7 
+```
+
+The same process can be followed for other status updates, whether you wish to mark a vulnerability as “Not affected” in the case of a [false positive finding](https://edu.chainguard.dev/chainguard/chainguard-images/recommended-practices/false-results/), “Fix not planned”, or another applicable status. 
+
+## Further Reading
+
+In this guide, you learned how to use the `wolfictl` tool to interact with Chainguard’s Security Advisories feed. You used `wolfictl` to explore existing advisories, and also created and updated your new security advisory. The steps shown in this guide allowed you to make local changes to your advisory feed. If you wish to contribute to the open-source Wolfi OS advisory feed, please read through our [How To Patch CVEs](
+https://github.com/wolfi-dev/os/blob/main/HOW_TO_PATCH_CVES.md) guide and our [How Chainguard Issues Security Advisories](^^^ LINK TO OTHER ARTICLE ^^^) article first.
```
Comment (luhring) [inline docs/managing-advisories.md:145]: What do you think the best way to handle this is? 🤔

**Review by luhring — COMMENTED** [review]
This is awesome @mcaveety, thank you! A couple quick things and then let's merge this!

File: docs/managing-advisories.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
```
Comment (mcaveety) [inline docs/managing-advisories.md:145]:
```suggestion
https://github.com/wolfi-dev/os/blob/main/HOW_TO_PATCH_CVES.md) guide first.
```

File: docs/managing-advisories.md (hunk 2 above)
Comment (luhring) [reply docs/managing-advisories.md]: Okay cool!

**Review by luhring — APPROVED** [review]

**Pull request merged**